### DISCUSSION TITLE
Accidentally dismissed input method init prompt, how can I enable input method manually?

### DISCUSSION THREAD

**Discussion by Butanediol** [shedaniel/RoughlyEnoughItems#1296] [Q & A] [answered]
This may sound dumb but I didn't find this settings in RoughlyEnoughItems dashboard.

**Comment by shedaniel** [comment, marked as answer]
hover the craftable filter button on the left of the config button

**Reply by Butanediol** [reply]
Wow this is fast. Thank you very much!
